pinot-core/src/main/java/org/apache/pinot/core/data/recordtransformer/PinotDataType.java

```diff
@@ -470,6 +474,43 @@ public String toString(Object value) {
     }
   },
 
+  HASHMAP,
```

Comment:
We actually don't have to introduce this type (in future we should when we have Map) but now there is no need. 

In the transform() method of DataTypeTransformer, we should do:

```
if (values[0] instance of Map) {
 // and if schema says the column is primitive
// then get the primitive type from Map using the function you have already implemented `getPinotDataTypeFromHashMap`
call dest.convert()
}
```